packages/fxa-graphql-api/src/gql/model/subscription.ts

```diff
@@ -5,7 +5,7 @@ import { Field, ObjectType } from '@nestjs/graphql';
 
 @ObjectType()
 export class Subscription {
-  @Field()
+  @Field({ nullable: true })
```

Comment:
For this field only, if I don't do this for an IAP subscription returned by the /account route (which does not have a `created` field) , I get a `"Error: Cannot return null for non-nullable field Subscription.created."` in the GraphQL API response in FxA Settings.